uf3/forcefield/calculator.py

```diff
@@ -111,31 +128,35 @@ def coefficients(self):
     def chemical_system(self):
         return self.bspline_config.chemical_system
 
+
     def get_potential_energy(self,
                              atoms: ase.Atoms = None,
                              force_consistent: bool = None
                              ) -> float:
         """Evaluate the total energy of a configuration."""""
-        if any(atoms.pbc):
-            supercell = geometry.get_supercell(atoms, r_cut=self.r_cut)
-        else:
-            supercell = atoms
-
-        if force_consistent is not True:
-            e_1b = self.energy_1b(atoms)
-        else:
-            e_1b = 0.0
-
-        if self.degree >= 2:
-            e_2b = self.energy_2b(atoms, supercell)
+        if atoms == None:
```

Comment:
```suggestion
        if atoms is None:
```
Using the `is` operator to check for `None` values [is preferred](https://stackoverflow.com/questions/3257919/what-is-the-difference-between-is-none-and-none) over `==`.